### PR TITLE
Create BMP.md

### PR DESCRIPTION
**Pull request by brmcdoug**
I've gotten a start on a BMP.md doc.  Does this format work and give you what you're looking for?

### CODE REVIEW

File: BMP.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# BGP Monitoring Protocol (BMP)
+
+https://tools.ietf.org/html/rfc7854
+
+## BMP message types and GoBMP:
```
Comment (sbezverk) [inline BMP.md]: When displayed as reach text, it is bit hard to read as everything is displayed on the same line. Could you please check the formatting?

File: BMP.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+GoBMP will not need to collect and parse the following BMP message types:
```
Comment (sbezverk) [inline BMP.md:6]: In both statements you say GoBMP will **not**, I suspect one of these should be without "not" right?

File: BMP.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+l3vpn
+evpn
+```
+With much gratitude we are using OpenBMP's message parsing as a template:
+
+https://www.snas.io/docs/
+
+#### BMP peer message:
+```
+1: (action): up/down
```
Comment (sbezverk) [inline BMP.md]: Please confirm if `action` field is a string with actual strings "up" or "down" or it is numerical enumeration where for example 0 means up and 1 means down.

File: BMP.md (hunk 2 above)
Comment (brmcdoug) [reply BMP.md:6]: fixed, thanks

**Pull request merged**